### DISCUSSION TITLE
Accessibility Statement

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#36] [Added]
### Type name

Accessibility Statement

### Alternative type names

~Accessibility~ https://github.com/OpenTermsArchive/terms-types/discussions/36#discussioncomment-10190157
Accessibility Policy

### Writer

service provider

### Audience

~end user~
everyone https://github.com/OpenTermsArchive/terms-types/discussions/36#discussioncomment-10199165

### Object

accessibility of the service to people with a disability

### Examples

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | impots.gouv.fr | [URL 🔗](https://www.impots.gouv.fr/accessibilite-des-services-en-ligne-des-professionnels-non-conforme) | DÉCLARATION D’ACCESSIBILITÉ DES SERVICES EN LIGNE | 🇫🇷 |
| 2 | Chambre des députés du Luxembourg | [URL 🔗](https://www.chd.lu/fr/declaration-daccessibilite)  | Déclaration d’accessibilité | 🇱🇺 |
| 3 | Burger King | [URL 🔗](https://www.bk.com/accessibility) | Accessibility | - |
| 4 | eSafety | [URL 🔗](https://www.esafety.gov.au/about-us/corporate-documents/website-accessibility) | Website accessibility | 🇦🇺 |
| 5 | IPVanish | [URL 🔗](https://www.ipvanish.com/accessibility/) | Accessibility Statement | - |
| 6 | ABC | [URL 🔗](https://www.abc.net.au/about/accessibility-statement/103700726) | Accessibility Statement | 🇦🇺 |
| 7 | www.gov.uk | [URL 🔗](https://www.gov.uk/help/accessibility-statement) | Accessibility statement | 🇬🇧 |
 
### References

- [Loi n°2005-102 du 11 février 2005 pour l'égalité des droits et des chances, la participation et la citoyenneté des personnes handicapées, article 47](https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000001290363)
- [Public Sector Bodies Accessibility Regulations 2018/2022](https://www.legislation.gov.uk/uksi/2018/952/made)
- [Loi du 28 mai 2019 relative à l’accessibilité des sites internet et des applications mobiles des organismes du secteur public](https://legilux.public.lu/eli/etat/leg/loi/2019/05/28/a373/jo)

**Comment by MattiSG** [comment]
## Examples

- `GOV.UK` should be `www.gov.uk`. Indeed:

> This accessibility statement applies to the www.gov.uk website. It does not cover other GOV.UK subdomains such as service subdomains

### Additional examples

- impots.gouv.fr | https://www.impots.gouv.fr/accessibilite-des-services-en-ligne-des-professionnels-non-conforme | DÉCLARATION D’ACCESSIBILITÉ DES SERVICES EN LIGNE | 🇫🇷
- Chambre des députés du Luxembourg | https://www.chd.lu/fr/declaration-daccessibilite | Déclaration d’accessibilité | 🇱🇺

## Alternative names

I don't believe that “Accessibility Policy” is a relevant type name, as these terms do not seem to set forth a policy but rather describe a current state, processes, and rights of end users.
“Accessibility” is not relevant either as it does not add information compared to “Accessibility Statement”.

## Object

I suggest the object to be more precise than “service accessibility”: “accessibility of the service to people with a disability”.

## References

- [Loi n°2005-102 du 11 février 2005 pour l'égalité des droits et des chances, la participation et la citoyenneté des personnes handicapées, article 47](https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000001290363)
- [Public Sector Bodies Accessibility Regulations 2018/2022](https://www.legislation.gov.uk/uksi/2018/952/made)
- [Loi du 28 mai 2019 relative à l’accessibilité des sites internet et des applications mobiles des organismes du secteur public](https://legilux.public.lu/eli/etat/leg/loi/2019/05/28/a373/jo)

**Comment by MattiSG** [comment]
## Audience

In this case, the audience is probably more than end users: auditors could also be interested, or people with a disability even when they don't use the service themselves. I would thus suggest that, like with “Imprint”, we use “everyone“ as the audience.

**Comment by clementbiron** [comment]
This terms type suggestion has been accepted and will soon be added.
